src/Discord.Net.Core/Discord.Net.Core.csproj

```diff
@@ -8,10 +8,10 @@
     <TargetFrameworks Condition=" '$(OS)' != 'Windows_NT' ">netstandard2.0;netstandard2.1</TargetFrameworks>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Newtonsoft.Json" Version="12.0.2" />
-    <PackageReference Include="System.Collections.Immutable" Version="1.3.1" />
-    <PackageReference Include="System.Interactive.Async" Version="4.0.0" />
-    <PackageReference Include="IDisposableAnalyzers" Version="2.1.2">
+    <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
+    <PackageReference Include="System.Collections.Immutable" Version="1.7.1" />
+    <PackageReference Include="System.Interactive.Async" Version="4.1.1" />
```

Comment:
`System.Interactive.Async` should not be referenced anymore, it overwrites `IAsyncEnumerable` references and causes problems. `IAsyncEnumerable` is loaded from `Microsoft.Bcl.AsyncInterfaces`.

Reply:
So Discord.net should not reference it at all?

Reply:
This is false - System.Interactive.Async is fine to reference, since it depends on System.Linq.Async which depends on Microsoft.Bcl.AsyncInterfaces as necessary.

Reply:
Indeed D.NET should not reference it. It's causing Easily googled wide spread issues, e.g. https://github.com/dotnet/efcore/issues/15047

TLDR referencing this is obsolete, it should **no longer** be referenced. I guess it becomes a question of what's the lowest supported version (and only reference it if really necessary)

Reply:
Odd I never had issues like that before on Windows. Maybe because you use a different platform?

For me it just uses the highest version of the assemblies even if their are other assemblies that calls for an older version of it.

Reply:
@RheaAyase ok, I checked further and it seems to work on that package since .NET Core 2.1 with the 5.0.0 version of them.

However oddly enough some 5.0.0 packages on my bot not the case, I also think I should add ``net5.0`` target frameworks to Discord.NET since I would need that target framework on my bot and for those users who wants to use net5.0 as well for updated things like nullable support.